crates/ruff/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -494,6 +550,31 @@ fn check_values(checker: &mut Checker, names: &Expr, values: &Expr) {
     }
 }
 
+fn find_duplicates(elts: &[Expr]) -> Vec<(usize, usize)> {
+    let mut duplicates: Vec<(usize, usize)> = Vec::new();
```

Comment:
Should we use a `SmallVec` since it's unlikely for there to be many duplicates?

Reply:
I tend to prefer `Vec` unless we can demonstrate that `SmallVec` has better performance. In this case, we probably don't expect _any_ duplicates in general, in which case this will never allocate anyway, so seems okay in my opinion.